spec/rubocop/factory_bot/config_formatter_spec.rb

```diff
@@ -40,28 +40,28 @@
   it 'builds a YAML dump with spacing between cops' do
     formatter = described_class.new(config, descriptions)
 
-    expect(formatter.dump).to eql(<<-YAML.gsub(/^\s+\|/, ''))
-      |---
-      |AllCops:
-      |  Setting: forty two
-      |
-      |FactoryBot/Foo:
-      |  Config: 2
-      |  Enabled: true
-      |  Description: Blah
-      |  Reference: https://www.rubydoc.info/gems/rubocop-factory_bot/RuboCop/Cop/FactoryBot/Foo
-      |
-      |FactoryBot/Bar:
-      |  Enabled: true
-      |  Nullable: ~
-      |  Description: Wow
-      |  Reference: https://www.rubydoc.info/gems/rubocop-factory_bot/RuboCop/Cop/FactoryBot/Bar
-      |
-      |FactoryBot/Baz:
-      |  Enabled: true
-      |  StyleGuide: "#buzz"
-      |  Description: Woof
-      |  Reference: https://www.rubydoc.info/gems/rubocop-factory_bot/RuboCop/Cop/FactoryBot/Baz
+    expect(formatter.dump).to eql(<<~YAML.gsub(/^\s+\|/, ''))
```

Comment:
`gsub` looks unnecessary:

```suggestion
    expect(formatter.dump).to eql(<<~YAML)
```